Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift

```diff
@@ -4,13 +4,11 @@
 
 // Disabled: It will be updated in FXIOS-6128
 
-/*
 import Foundation
 import XCTest
 import SwiftUI
 @testable import Client
 
-
 class CreditCardInputFieldTests: XCTestCase {
     @State var testableString: String = ""
```

Comment:
Do you need the state var still? 🤔 
Is it possible to test without using state variables

Reply:
Updated - no longer using this!